engine/schema/src/main/resources/META-INF/db/schema-41910to42000.sql

```diff
@@ -425,3 +425,8 @@ INSERT IGNORE INTO `cloud`.`guest_os_hypervisor` (uuid, hypervisor_type, hypervi
 
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.vm_instance', 'delete_protection', 'boolean DEFAULT FALSE COMMENT "delete protection for vm" ');
 CALL `cloud`.`IDEMPOTENT_ADD_COLUMN`('cloud.volumes', 'delete_protection', 'boolean DEFAULT FALSE COMMENT "delete protection for volumes" ');
+
+-- Create a new group for Usage Server related configurations
+INSERT INTO `cloud`.`configuration_group` (`name`, `description`, `precedence`) VALUES ('Usage Server', 'Usage Server related configuration', 9);
```

Comment:
cc @sureshanaparti to check if configkeys can be self-discovered without any schema paths? Otherwise, we may need to move this to a new 4.20.0->4.20.1 upgrade path.

Reply:
ping @sureshanaparti

Reply:
yes, the config keys will be discover group and sub-group if they doesn't have. If config keys already have and need to be re-discover to this, then schema path needs to remove group and sub-group for the configs Or update configs with group and sub-group ids with the new config group and sub-group ids.

cc @abh1sar @borisstoyanov @rohityadavcloud